CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Lotus changelog
 
+# 0.7.1 / 2020-09-17
+
+This optional release of Lotus upgrades some core dependencies, and introduces many improvements to the mining process, deal-making cycle, and overall User Experience.
+
```

Comment:
- Consistently name winning and window post in logs (https://github.com/filecoin-project/lotus/pull/3873)

Reply:
```suggestion
Some notable improvements: 
- Consistently name winning and window post in logs ([#3873](https://github.com/filecoin-project/lotus/pull/3873))
- Add auto flag to mpool replace ([#3752](https://github.com/filecoin-project/lotus/pull/3752))
- Update go-fil-markets to 0.6.1 ([changelog](https://github.com/filecoin-project/go-fil-markets/blob/v0.6.1/CHANGELOG.md#go-fil-markets-061)
```

Reply:
* Journal stuff